### PR TITLE
Fix injection problems in the workspace sidebar

### LINKED ISSUE #413: as a user in a workspace i can see all members when clicking nothing happens

### PR DESCRIPTION
**Pull request by ale-rt**
Fixes #413

### CODE REVIEW

File: src/ploneintranet/suite/tests/acceptance/workspace.robot [hunk 1]
```diff
@@ -32,13 +32,14 @@ Breadcrumbs are not borked
...
 
-Manager can view sidebar info
-    Given I am in a workspace as a workspace admin
-    I can go to the sidebar info tile
-
 Member can view sidebar info
     Given I am in a workspace as a workspace member
-    I can go to the sidebar info tile
+          I can go to the sidebar info tile
+          Click Link  Members
```
Comment (ale-rt) [inline src/ploneintranet/suite/tests/acceptance/workspace.robot]: Anybody is welcome to create a proper keyword for this test :)

**Comment by gyst** [comment]
The user profiles loaded with injection this way are not fully functional. If you follow/unfollow somebody the @@toggle_follow view does not change the relationship. In a standalone profile view this does work.

**Comment by ale-rt** [comment]
@gyst this is the relevant commit:
- https://github.com/ploneintranet/ploneintranet/commit/ec52796acce4a0e343b9206bf2f5307216d81e60
  The robot tests are ugly by they pass

If anybody can take care of making them beautiful I will be glad!

**Comment by gyst** [comment]
Another unrelated Heisenbug. Jenkins test this please test

**Comment by gyst** [comment]
Failure is a damn Heisenfail. Jenkins test this please test.
Works on my machine.

**Comment by gyst** [comment]
Finally green!

**Pull request merged**